Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM node:8-alpine as build
+RUN apk add --no-cache git
```

Comment:
Is installing Git really needed?

Reply:
Good catch. No it is not, must be a left over from previous versions. I will remove it.